dockerfile/rocm4.2-pytorch1.7.0.dockerfile

```diff
@@ -41,6 +42,18 @@ RUN wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | APT_KEY_DON
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/*
 
+# Install Open MPI
+ENV OPENMPI_VERSION=4.0.5
+RUN cd /tmp && \
+    wget -q https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-${OPENMPI_VERSION}.tar.gz && \
+    tar xzf openmpi-${OPENMPI_VERSION}.tar.gz && \
+    cd openmpi-${OPENMPI_VERSION} && \
+    ./configure --enable-orterun-prefix-by-default && \
+    make -j $(nproc) all && \
+    make install && \
+    ldconfig && \
+    rm -rf openmpi-${OPENMPI_VERSION}*
```

Comment:
Do we need to do `cd ..` first?

Reply:
Good catch

Reply:
change to `rm -rf /tmp/openmpi-${OPENMPI_VERSION}*`